vnext/Test/React.Windows.Test.vcxproj

```diff
@@ -60,7 +60,7 @@
       -->
       <PreprocessorDefinitions>
         BOOST_ASIO_HAS_IOCP;
-        WINVER=$(WinVer);
+        _WIN32_WINNT=$(WinVer);
         %(PreprocessorDefinitions)
```

Comment:
We don't define WIN32 here. Do we need this preprocessor definition?

Reply:
It was there before, so it does not affect (it does not transitively define `WIN32` or anything).
Being a shared static lib, I may be able to leave it out.